CMake/HPHPSetup.cmake

```diff
@@ -60,7 +60,12 @@ if(HHVM_DYNAMIC_EXTENSION_DIR)
   add_definitions(-DHHVM_DYNAMIC_EXTENSION_DIR="${HHVM_DYNAMIC_EXTENSION_DIR}")
 else()
   if(UNIX)
-    add_definitions(-DHHVM_DYNAMIC_EXTENSION_DIR="${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/hhvm/extensions")
+    if(IS_ABSOLUTE "${CMAKE_INSTALL_LIBDIR}")
+      set(HHVM_DYNAMIC_EXTENSION_DIR "${CMAKE_INSTALL_LIBDIR}/hhvm/extensions")
```

Comment:
Since this is using [`GNUInstallDirs`](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html), you can just switch to `CMAKE_INSTALL_FULL_LIBDIR` to get an absolute path.

Reply:
Fixed in #9085